nix/tests.nix

```diff
@@ -97,7 +98,7 @@ let
 
         ${cabal-install}/bin/cabal v2-build ${devCabalOptions}
         ${cabal-install}/bin/cabal v2-exec ${withTmpDb postgresql} \
-          ${ioTestPython}/bin/pytest -- -v "$rootdir"/test/io-tests "$@"
```

Comment:
I think we can skip a few of those `$rootdir`s, because we are doing `cd $rootdir` in those scripts.